extensions/adobe/experience/decisioning/placement.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/decisioning/placement",
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "type": "object",
+  "title": "Content Placement",
+  "description": "A content placement describes a location or place where the content of a proposed decision option is rendered into. It is used to set technical constraints for content that the decision supplies.",
+  "definitions": {
+    "placement": {
+      "properties": {}
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/data/record"
+    },
+    {
+      "$ref": "#/definitions/placement"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/decisioning/placement-details#/definitions/placement-details"
```

Comment:
Seems like, placement-details would always be used with placement as the name signifies. Should "placement-details " even be a data type ? Can we add those properties here  ?

Reply:
In several other snapshots placement-details are captured within those datatypes, often in arrays. We need that reusable field set. But what could be done for all the entity types is the following. They could absorb the definitions, keep them separate and when another composite datatype needs them we can go after the -details definition of the entity schema. The downside is that it makes it impossible for custom schemas to reuse those definitions because they will not become a bonafide datatype that can be reused at the logical level. The file and "definitions" structure is invisible after the registry has composed the schemas